README.md

```diff
@@ -191,13 +227,49 @@ To run, build and test locally:
     * `GOVUK_NOTIFY_TEMPLATE_ID`: the GovUK Notify service template id secret value to be used
     * `GOVUK_NOTIFY_API_KEY`: the GovUK Notify service api key secret value to be used
     * `DB_VENDOR`: the type of database to use. In the dev environment, we use Keycloak's embedded H2 database
+    * `SNS_TOPIC_ARN`: the AWS topic arn to publish event messages to
+    * `TDR_ENV`: the name of the TDR environment where Keycloak is running
 6. Navigate to http://localhost:8081/auth/admin
 7. Log on using the `KEYCLOAK_PASSWORD` and `KEYCLOAK_USER` defined in the docker run command
 
 To log into the running docker container with a bash shell: `$ docker exec -it [your container name] bash`
 
 Make changes to the realm export json file as necessary to test new configurations.
 
+### Optionally Run Event Publishing to AWS SNS Topic
+
+If you are working on the event publishing you can optionally send messages to an AWS SNS topic.
+
+To run the event publishing:
+
+1. When setting the environment variables for running the local docker image ensure the `SNS_TOPIC_ARN` value is the ARN of the topic you wish to publish to.
+2. Once the local Keycloak container is running log into it: `$ docker exec -it [your container name] bash`
```

Comment:
The other way to do this is to set the credentials in the environment variables. If you have the environment variables set locally then you can do it like
`-e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID  -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -e AWS_SESSION_TOKEN=$AWS_SESSION_TOKEN`

It's not much easier and you still have to stop and restart the container when the credentials expire but you don't have to `docker exec` in. 
It's up to you. You can add this as an alternative method or just ignore me.

Reply:
Yeah, good point. I've added those as an alternative method.